anisoap/representations/radial_basis.py

```diff
@@ -272,3 +452,36 @@ def orthonormalize_basis(self, features: TensorMap):
                 )
 
         return features
+
+    def get_basis(self, rs):
+        from matplotlib import pyplot as plt
+
+        all_gs = np.empty(shape=(len(rs), 1))
+        for l in range(0, self.max_angular):
+            n_arr = np.arange(self.num_radial_functions[l])
+            l_2n_arr = l + 2 * n_arr
```

Comment:
This is something I want to clarify (and I know this is what I did too in orthonormalization). when we generate a set of basis functions, are we only doing basis functions of order l+2narr, or are we generating all orders from 0 to maxdeg=maxl + maxn  and then filtering the l+2narr columns and rows? This function and my orthonormalization function is doing the latter, I just want to make sure that's correct.

Reply:
This is a longer discussion that can go into an issue

Reply:
ok